pipeline_steps/common.groovy

```diff
@@ -656,28 +656,201 @@ def delete_workspace() {
   }
 }
 
-def create_jira_issue(String project="RE",
-                      String tag=env.BUILD_TAG,
-                      String link=env.BUILD_URL,
-                      String type="Issue-releng-platform-alert"){
+// This function creates or updates issues realted to build failures.
+// Should be used to report build failures by calling:
+//    common.build_failure_issue(project)
+void build_failure_issue(String project){
```

Comment:
Is this supposed to be `String` rather than `Void`?  Do we even need to return anything here?

Reply:
Should be String, may be useful to know the key of the issue that was found/created. Done